userInterface.js

```diff
@@ -1,27 +1,43 @@
 var blessed = require('blessed');
+var colorPal = require('./themes.js');
+var supportsColor = require('supports-color');
 
+// Colors
+var colorSupportLevel = supportsColor ? supportsColor.level : 0;
+var defaultThemesByColorLevel = {
+    3: colorPal.default,
+    2: colorPal.default, // Not tested yet
+    1: colorPal.tty16Default,
+    0: colorPal.tty1Default // Can be "tested" by passing --no-color argument
+};
+
+var theme = defaultThemesByColorLevel[colorSupportLevel];
+var colors = theme.palette;
```

Comment:
Can the above (L6 - L15) be pulled into another module to import? Also, I think it would be clearer to use the `supportsColor.has256`, `supportsColor.has16m`, etc. rather than doing it by level.